spec/requests/api/v2/session_spec.rb

```diff
@@ -106,16 +106,16 @@
         end
       end
 
-      context 'on first login' do
-        let(:user) { create(:user, first_login_date_utc: nil) }
+      context 'When the user has not received a welcome email' do
+        let(:user) { create(:user, welcome_email_date_utc: nil) }
```

Comment:
```suggestion
        let(:user) { create(:user, :first_timer) }
```

maybe add a trait in factory that nils out the `first_login_date_utc` and the `welcome_email_date_utc` fields

Reply:
Maybe if it becomes more useful outside of these two instances - but I find traits can make it less clear as to what's being tested sometimes